packages/gatsby-source-kontent/src/client.ts

```diff
@@ -12,18 +12,21 @@ import {
   version as packageVersion,
 } from '../package.json';
 
-const KontentDeliveryProductionDomain = 'https://deliver.kontent.ai';
-const KontentDeliveryPreviewDomain = 'https://preview-deliver.kontent.ai';
+const DefaultKontentDeliveryProductionDomain = 'deliver.kontent.ai';
+const DefaultKontentDeliveryPreviewDomain = 'preview-deliver.kontent.ai';
 const continuationHeaderName = 'x-continuation';
 const authorizationHeaderName = 'authorization';
 const trackingHeaderName = 'x-kc-source';
+const waitForLoadingNewContentHeaderName = 'x-kc-wait-for-loading-new-content';
 
 rax.attach();
 
-const getDomain = (options: CustomPluginOptions): string =>
-  options.usePreviewUrl
-    ? KontentDeliveryPreviewDomain
-    : KontentDeliveryProductionDomain;
+const getProtocolAndDomain = (options: CustomPluginOptions): string => {

```

Comment:
I'd stick with the previous `getDomain` or `getDeliveryDomain`. Delivery service will be running probably on https for quite a long time.

Reply:
OK - I found on Wikipedia that domain is without protocol :-) 

How about getLeftPart()? 
https://stackoverflow.com/questions/2081645/what-do-you-call-the-entire-first-part-of-a-url

Reply:
Agreed to leave it like that.